widgetastic_manageiq/__init__.py

```diff
@@ -2078,8 +2254,18 @@ def entity_names(self):
 
         Returns: all current page entities
         """
-        br = self.browser
-        return [br.get_attribute('title', el) for el in br.elements(self.elements)]
+        current_version = VersionPick({
+            Version.lowest(): 'old',
+            '5.9': 'new',
+            'upstream': 'new'
+        }).pick(self.browser.product_version)
```

Comment:
VersionPick is designed to be used on classes, if you have only two values and a decision in code, this is much shorter and more readable.